manifests/dotnet.yml

```diff
@@ -27,7 +27,8 @@ tests/:
         test_ldap_injection.py:
           TestLDAPInjection: missing_feature
         test_no_httponly_cookie.py:
-          TestNoHttponlyCookie: missing_feature
+          TestNoHttponlyCookie:
```

Comment:
Not a request change, just a comment for later PR, you cam simply do that : 

`TestNoHttponlyCookie: v2.39.0`

The `*` is handy when you have different version for different weblogs.

Reply:
Thanks! I will create a separate PR and simplify all the existing test that we already have in IAST like that.